poly-26-02/svg_2_css.py

```
@@ -0,0 +1,40 @@
+import xml.etree.ElementTree as ET
+
+def extract_fill_colors(svg_file):
+    """Extracts unique fill colors from an SVG file."""
+    fill_colors = set()
+    try:
+        tree = ET.parse(svg_file)
```

Ruff (0.14.14)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)